packages/react-native-bridge/index.js

```diff
@@ -262,16 +262,6 @@ export function mediaUploadSync() {
 	return RNReactNativeGutenbergBridge.mediaUploadSync();
 }
 
-/**
- * Request to start listening to save events when in-progress saves are in place
- *
- * For example, when media is being saved and the user re-enters the editor
- *
- */
-export function mediaSaveSync() {
-	return RNReactNativeGutenbergBridge.mediaSaveSync();
-}
```

Comment:
Only used in Story block ([reference](https://github.com/Automattic/jetpack/pull/36151/files#diff-3ec47af4510e547616f9dbcdcd2557a8cd539dcaf938c8b1c1a7d5f42fc696a7L43)).